app/controllers/movies_controller.rb

```diff
@@ -143,23 +141,23 @@ def set_movie
     @movie = Movie.find(params[:id])
   end
 
-  def set_user
-    @user = current_user
+  def set_movie_review
+    @reviews = @user.reviews
+    @movie_review = @reviews.where(movie_id: @movie.id).first
```

Comment:
@sdsunjay change this to `@reviews.find_by(movie_id: @movie.id)`

Reply:
Does it make a difference? is it faster to do it like that?

Reply:
No `first` necessary?!

Reply:
@lifeiscontent?